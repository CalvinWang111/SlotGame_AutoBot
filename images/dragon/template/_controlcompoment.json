{
    "stickers": {
        "path": "D:\\git\\SlotGame_AutoBot\\images\\dragon\\segmented_image\\cropped_image_33_contour_2.png",
        "confidence": 0.965180516242981,
        "contour": [
            406,
            296,
            3,
            8
        ],
        "value": null
    },
    "button_start_spin": {
        "path": "D:\\git\\SlotGame_AutoBot\\images\\dragon\\segmented_image\\cropped_image_10_contour_0.png",
        "confidence": 0.7519719004631042,
        "contour": [
            425,
            948,
            89,
            63
        ],
        "value": null
    },
    "button_max_bet": {
        "path": "D:\\git\\SlotGame_AutoBot\\images\\dragon\\segmented_image\\cropped_image_47_contour_0.png",
        "confidence": 0.4994920790195465,
        "contour": [
            364,
            949,
            54,
            31
        ],
        "value": null
    },
    "button_three_dot": {
        "path": "D:\\git\\SlotGame_AutoBot\\images\\dragon\\segmented_image\\cropped_image_43_contour_0.png",
        "confidence": 0.4975583553314209,
        "contour": [
            498,
            96,
            47,
            48
        ],
        "value": null
    },
    "button_speedup_spin": {
        "path": "D:\\git\\SlotGame_AutoBot\\images\\dragon\\segmented_image\\cropped_image_46_contour_0.png",
        "confidence": 0.31504562497138977,
        "contour": [
            364,
            979,
            54,
            32
        ],
        "value": null
    },
    "button_increase_bet": {
        "path": "D:\\git\\SlotGame_AutoBot\\images\\dragon\\segmented_image\\cropped_image_48_contour_0.png",
        "confidence": 0.5917772054672241,
        "contour": [
            162,
            948,
            26,
            64
        ],
        "value": null
    },
    "button_decrease_bet": {
        "path": "D:\\git\\SlotGame_AutoBot\\images\\dragon\\segmented_image\\cropped_image_51_contour_0.png",
        "confidence": 0.5950870513916016,
        "contour": [
            38,
            949,
            25,
            62
        ],
        "value": null
    },
    "button_additional_bet": {
        "path": "D:\\git\\SlotGame_AutoBot\\images\\dragon\\segmented_image\\cropped_image_37_contour_1.png",
        "confidence": 0.3922345042228699,
        "contour": [
            50,
            924,
            131,
            27
        ],
        "value": null
    }
}