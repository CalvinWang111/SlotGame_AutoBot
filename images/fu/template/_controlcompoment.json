{
    "stickers": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\fu\\segmented_image\\cropped_image_99_contour_0.png",
        "confidence": 0.9651821851730347,
        "contour": [
            56,
            168,
            18,
            40
        ],
        "value": null
    },
    "button_info": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\fu\\segmented_image\\cropped_image_29_contour_0.png",
        "confidence": 0.583082377910614,
        "contour": [
            155,
            795,
            100,
            119
        ],
        "value": null
    },
    "button_max_bet": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\fu\\segmented_image\\cropped_image_53_contour_0.png",
        "confidence": 0.5059353113174438,
        "contour": [
            336,
            869,
            49,
            29
        ],
        "value": null
    },
    "button_increase_bet": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\fu\\segmented_image\\cropped_image_52_contour_0.png",
        "confidence": 0.6022526025772095,
        "contour": [
            149,
            868,
            25,
            60
        ],
        "value": null
    },
    "button_decrease_bet": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\fu\\segmented_image\\cropped_image_39_contour_0.png",
        "confidence": 0.5880782604217529,
        "contour": [
            273,
            799,
            51,
            113
        ],
        "value": null
    },
    "button_three_dot": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\fu\\segmented_image\\cropped_image_51_contour_0.png",
        "confidence": 0.45921605825424194,
        "contour": [
            458,
            83,
            43,
            44
        ],
        "value": null
    },
    "button_home": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\fu\\segmented_image\\cropped_image_43_contour_0.png",
        "confidence": 0.6335330009460449,
        "contour": [
            92,
            38,
            73,
            74
        ],
        "value": null
    },
    "button_start_spin": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\fu\\segmented_image\\cropped_image_10_contour_0.png",
        "confidence": 0.7411547899246216,
        "contour": [
            391,
            869,
            82,
            58
        ],
        "value": null
    },
    "button_speedup_spin": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\fu\\segmented_image\\cropped_image_53_contour_0.png",
        "confidence": 0.35394802689552307,
        "contour": [
            336,
            898,
            49,
            29
        ],
        "value": null
    }
}