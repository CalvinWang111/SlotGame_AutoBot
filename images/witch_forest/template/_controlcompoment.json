{
    "stickers": {
        "path": "c:\\Users\\pppoo\\Desktop\\SlotGame_AutoBot\\images\\witch_forest\\segmented_image\\cropped_image_46_contour_1.png",
        "confidence": 0.9651671051979065,
        "contour": [
            13,
            9,
            29,
            23
        ],
        "value": null
    },
    "button_info": {
        "path": "c:\\Users\\pppoo\\Desktop\\SlotGame_AutoBot\\images\\witch_forest\\segmented_image\\cropped_image_12_contour_0.png",
        "confidence": 0.5738906860351562,
        "contour": [
            175,
            876,
            116,
            138
        ],
        "value": null
    },
    "button_start_spin": {
        "path": "c:\\Users\\pppoo\\Desktop\\SlotGame_AutoBot\\images\\witch_forest\\segmented_image\\cropped_image_4_contour_0.png",
        "confidence": 0.7407245635986328,
        "contour": [
            1305,
            875,
            271,
            139
        ],
        "value": null
    },
    "button_increase_bet": {
        "path": "c:\\Users\\pppoo\\Desktop\\SlotGame_AutoBot\\images\\witch_forest\\segmented_image\\cropped_image_26_contour_0.png",
        "confidence": 0.58659827709198,
        "contour": [
            592,
            878,
            66,
            126
        ],
        "value": null
    },
    "button_decrease_bet": {
        "path": "c:\\Users\\pppoo\\Desktop\\SlotGame_AutoBot\\images\\witch_forest\\segmented_image\\cropped_image_30_contour_0.png",
        "confidence": 0.5771519541740417,
        "contour": [
            299,
            879,
            58,
            123
        ],
        "value": null
    },
    "button_home": {
        "path": "c:\\Users\\pppoo\\Desktop\\SlotGame_AutoBot\\images\\witch_forest\\segmented_image\\cropped_image_33_contour_0.png",
        "confidence": 0.6375995874404907,
        "contour": [
            99,
            46,
            80,
            81
        ],
        "value": null
    },
    "button_three_dot": {
        "path": "c:\\Users\\pppoo\\Desktop\\SlotGame_AutoBot\\images\\witch_forest\\segmented_image\\cropped_image_37_contour_0.png",
        "confidence": 0.4880300462245941,
        "contour": [
            1651,
            48,
            79,
            79
        ],
        "value": null
    },
    "button_max_bet": {
        "path": "c:\\Users\\pppoo\\Desktop\\SlotGame_AutoBot\\images\\witch_forest\\segmented_image\\cropped_image_8_contour_0.png",
        "confidence": 0.5020537972450256,
        "contour": [
            1094,
            875,
            194,
            130
        ],
        "value": null
    }
}