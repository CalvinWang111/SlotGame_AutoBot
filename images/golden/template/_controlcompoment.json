{
    "stickers": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\golden\\segmented_image\\cropped_image_8_contour_1.png",
        "confidence": 0.9652833938598633,
        "contour": [
            82,
            526,
            2,
            3
        ],
        "value": null
    },
    "button_start_spin": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\golden\\segmented_image\\cropped_image_16_contour_0.png",
        "confidence": 0.7411540150642395,
        "contour": [
            391,
            869,
            82,
            58
        ],
        "value": null
    },
    "button_three_dot": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\golden\\segmented_image\\cropped_image_66_contour_0.png",
        "confidence": 0.47296300530433655,
        "contour": [
            458,
            83,
            43,
            44
        ],
        "value": null
    },
    "button_max_bet": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\golden\\segmented_image\\cropped_image_69_contour_0.png",
        "confidence": 0.5095938444137573,
        "contour": [
            336,
            869,
            49,
            29
        ],
        "value": null
    },
    "button_increase_bet": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\golden\\segmented_image\\cropped_image_65_contour_0.png",
        "confidence": 0.5974780321121216,
        "contour": [
            149,
            869,
            24,
            59
        ],
        "value": null
    },
    "button_additional_bet": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\golden\\segmented_image\\cropped_image_47_contour_0.png",
        "confidence": 0.36232081055641174,
        "contour": [
            46,
            846,
            121,
            26
        ],
        "value": null
    },
    "button_decrease_bet": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\golden\\segmented_image\\cropped_image_72_contour_0.png",
        "confidence": 0.5923694968223572,
        "contour": [
            35,
            869,
            23,
            58
        ],
        "value": null
    },
    "button_speedup_spin": {
        "path": "c:\\NTUST\\GameLab\\Project\\Automatic_Slot_Bot\\SlotGame_AutoBot\\images\\golden\\segmented_image\\cropped_image_68_contour_0.png",
        "confidence": 0.35394802689552307,
        "contour": [
            336,
            898,
            49,
            29
        ],
        "value": null
    }
}